src/mongo/tree/MongoCollectionTreeItem.ts

```diff
@@ -173,8 +194,21 @@ export class MongoCollectionTreeItem implements IAzureParentTreeItem {
 			});
 	}
 
-	private insertMany(documents: any[]): Thenable<string> {
-		return this.collection.insertMany(documents)
+	private insertMany(args: any[]): Thenable<string> {
+		// documents = args[0], collectionWriteOptions from args[1]
+		let collectionWriteOptions = {};
+		if (args.length > 2) {
+			throw new Error("Too many arguments. Please see mongo shell documentation. https://docs.mongodb.com/manual/reference/method/db.collection.insertMany/#db.collection.insertMany");
+		} else if (args.length === 2) {
+			if (args[1] && args[1].ordered) {
+				collectionWriteOptions["ordered"] = args[1].ordered;
+			}
+			if (args[1] && args[1].writeConcern) {
+				collectionWriteOptions["writeConcern"] = args[1].writeConcern;
+			}
+		}
+
+		return this.collection.insertMany(args[0], collectionWriteOptions)
```

Comment:
Using `args[0]` feels ugly, but defining a variable for single use seems against practice.